src/package.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="formatstring" version="5.8.0" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="formatstring" version="6.0.1" xmlns="http://www.mendix.com/clientModule/1.0/">
```

Comment:
Wow 😂

Reply:
I was also surprised.. and in Marketplace it says 6, so cannot avoid this big mistake.